apps/dht/src/dht.c

```diff
@@ -0,0 +1,78 @@
+/* dht.c - KNoT Application Client */
+
+/*
+ * Copyright (c) 2019, CESAR. All rights reserved.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/*
+ * This example uses a DHT sensor to monitor ambient temperature and relative
+ * humidity.
```

Comment:
Explain here this example works with dht 11 and dht 22. The default is the dht 11 and the dht 22 can be used by changing the flags on prj.conf file.

Reply:
Done